.github/workflows/vitest.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Tests: vitest'
+
+on:
+  push:
+    branches: [ "main", "ragezbla/89-vitest-gh-actions" ] # TODO remove after checking it works
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '45 15 * * 4'
+
+jobs:
```

Comment:
This looks like an old job. It's talking about cypress, missing the job name, etc.

Reply:
@ermish not sure if I am following can you point out the actual syntax or doc you want to use?

Reply:
Sorry! I'll add specifics